cfme/tests/ssui/test_ssui_dashboard.py

```diff
@@ -15,14 +15,15 @@
 from cfme.utils.appliance import ViaSSUI
 from cfme.utils.wait import wait_for
 
-
+# Note that the tests are being collected for 5.8 only.
+# They can be collected for 5.7 and 5.9 once testing is done on those versions.
```

Comment:
**REQUIRED:** This comment doesn't make sense. They are not currently being collected for 5.8 only, what does it mean that they can be collected for 5.7 and 5.9 once testing is done on those versions? Surely the point of the automation is to test? I'm confused. Sorry